desktop/components/commercial_filter/filters/followed_artists/followed_artist_filter_view.coffee

```diff
@@ -29,8 +29,7 @@ module.exports = class FollowedArtistsFilterView extends Backbone.View
 
   signup: (e) ->
     e.preventDefault()
-    redirectTo = "#{window.location.pathname}#{window.location.search}"
-    mediator.trigger 'open:auth', mode: 'register', copy: "Sign up to receive alerts when new works are available by artists you follow.", redirectTo: redirectTo
+    mediator.trigger 'open:auth', mode: 'register', copy: "Sign up to receive alerts when new works are available by artists you follow."
```

Comment:
This passes in a `redirectTo` of the same page you're on. This has the effect of skipping onboarding entirely! Now, we accomplish the desired state: continue to onboarding, and after onboarding come back.